setup.py

```diff
@@ -1,14 +1,10 @@
 from setuptools import setup, find_packages  # pylint: disable=no-name-in-module,import-error
 
 
-def dependencies(file):
-    with open(file) as f:
-        return f.read().splitlines()
-
-
 setup(
     name='halo',
     packages=find_packages(exclude=('tests', 'examples')),
+    include_package_data=True,
```

Comment:
@theY4Kman What does this line do?

Reply:
lol, I'm not sure, tbh. The line was there previously, just below the reqs. I thought it looked out of place, so I moved it to the other line that said "package" :P

Reply:
Ah, just noticed! I should check but I remember stumbling upon an error before adding it.